### DISCUSSION TITLE
PoE Possible?

### DISCUSSION THREAD

**Discussion by lensherm** [tiny-pilot/tinypilot#668] [Q&A] [answered]
I was wondering if by powering using a PoE HAT is feasible, without the power backflow issue you mention.

**Comment by mtlynch** [comment, marked as answer]
The risk still exists with a PoE HAT. PoE HATs aren't designed with TinyPilot's use-case in mind so [they don't protect against incorrect power flows through the USB-C port](https://github.com/mtlynch/tinypilot/wiki/Powering-your-TinyPilot-safely#risks-with-poe).

I recently tried connecting a Pi + PoE HAT to a laptop from 2010 and it would instantly kill the laptop's power as soon as I connected the laptop's USB port to the Pi. It didn't seem like it did any permanent damage, but it's not the kind of thing I'd do regularly.

You could still do it safely if you combine a PoE HAT with a power connector (or some other means to remove the power line from the USB cable connecting the computer to the Pi).
